src/components/EntityScorecardsCard.tsx

```diff
@@ -0,0 +1,231 @@
+
+  return (
+    <InfoCard
+      title={
+        <Box
+          sx={{
+            display: "flex",
+            flexDirection: "column",
+            gridGap: 16,
+            marginBottom: -20,
+          }}
+        >
+          <BrandedCardTitle title="Scorecards" />
+          <Tabs
+            value={tab}
+            onChange={(_, value) => setTab(value)}
+            indicatorColor="primary"
+          >
+            <Tab value="levels" label="Levels" />
+            <Tab value="checks" label="Checks" />
+          </Tabs>
+        </Box>
+      }
+      deepLink={{
+        link: `https://app.getdx.com/catalog/${entityIdentifier}/scorecards`,
+        title: "View scorecards",
+      }}
+      variant="gridItem"
+    >
+      <Box sx={{ maxHeight: contentMaxHeight, overflow: "auto" }}>
+        {tab === "levels" && <LevelsTab scorecards={scorecards} />}
+
+        {tab === "checks" && <ChecksTab checks={flattenedChecks} />}
+      </Box>
+    </InfoCard>
+  );
+}
+
+function LevelsTab({ scorecards }: { scorecards: Scorecard[] }) {
+  return (
+    <Box
+      sx={{
+        display: "grid",
+        gridTemplateColumns: "minmax(25%, 3fr) minmax(25%, 2fr)",
+      }}
+    >
+      {scorecards.map((scorecard, idx) => (
+        <React.Fragment key={scorecard.id}>
+          <Box
+            sx={{
+              lineHeight: "40px",
+              fontWeight: 500,
+              fontSize: 13,
+              borderTop: idx === 0 ? "none" : `1px solid ${COLORS.GRAY_100}`,
+              paddingRight: 8,
+              overflow: "hidden",
+              textOverflow: "ellipsis",
+              whiteSpace: "nowrap",
+            }}
+          >
+            {scorecard.name}
+          </Box>
+          <Box
+            sx={{
+              display: "flex",
+              alignItems: "center",
+              fontSize: 13,
+              borderTop: idx === 0 ? "none" : `1px solid ${COLORS.GRAY_100}`,
+              whiteSpace: "nowrap",
+              color: "#616161",
+              minWidth: 0,
+            }}
+          >
+            <Box sx={{ mr: 1 }}>
+              <LevelIcon
+                color={
+                  scorecard.current_level?.color ??
+                  scorecard.empty_level.color ??
+                  DEFAULT_NO_LEVEL_COLOR
+                }
+              />
+            </Box>
+            <Box
+              sx={{
+                overflow: "hidden",
+                textOverflow: "ellipsis",
+                minWidth: 0,
+                whiteSpace: "nowrap",
+              }}
+            >
+              {scorecard.current_level?.name ??
+                scorecard.empty_level.label ??
+                "No level"}
+            </Box>
+          </Box>
+        </React.Fragment>
+      ))}
+    </Box>
+  );
+}
+
+function ChecksTab({ checks }: { checks: ScorecardCheck[] }) {
+  return (
+    <Box sx={{ display: "grid", gridTemplateColumns: "minmax(0, 3fr) 1fr" }}>
```

Comment:
We could maybe fiddle with these template columns a bit more for better responsiveness on very large / very small card widths?

Reply:
I think it's fine as-is. I'd rather let the renderer of the page change the card grid for responsiveness than this grid within the card.